src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstemplatepackagename/service/TemplateKotlinApiService.kt

```diff
@@ -4,24 +4,23 @@ import org.springframework.stereotype.Service
 import org.springframework.web.reactive.function.client.WebClient
 import org.springframework.web.reactive.function.client.WebClientResponseException
 import reactor.core.publisher.Mono
-import uk.gov.justice.hmpps.kotlin.auth.HmppsAuthenticationHolder
 
 @Service
 class TemplateKotlinApiService(
   private val templateKotlinApiWebClient: WebClient,
-  private val hmppsAuthenticationHolder: HmppsAuthenticationHolder,
```

Comment:
remove use of the holder - was causing confusion and shouldn't be needed for normal usage